ko/admin/config.rst

```diff
@@ -768,7 +768,7 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
     **db_volume_size**\ 는 다음과 같은 값을 설정하는 파라미터이다. 값 뒤에 B, K, M, G, T로 단위를 붙일 수 있으며, 각각 Bytes, Kilobytes, Megabytes, Gigabytes, Terabytes를 의미한다. 단위를 생략하면 바이트 단위가 적용된다. 기본값은 **512M**\ 이다.
 
     *   **cubrid createdb**\와 **cubrid addvoldb** 유틸리티에서 **\-\-db-volume-size** 옵션을 생략했을 때 생성되는 데이터베이스 볼륨의 기본 크기
-    *   데이터베이스 볼륨 공간을 모두 사용하면 자동으로 추가되는 볼륨의 기본 크기
+    *   데이터베이스 볼륨 공간을 모두 사용하면 자동으로 추가되는 볼륨의 최대 크기 (자동 추가 볼륨은 필요량만큼 점진적으로 증가되며 db_volume_size에 초과한 경우 다른 추가 볼륨으로 생성됨)
```

Comment:
How about adding a formula for the calculation for the new volume size?

Reply:
Thank you for comment.
But, In my opinion, it is better not to include information that can be changed by code modification in the manual.